web/src/app/store.ts

```diff
@@ -45,13 +45,14 @@ export function makeAillyStore(dispatch: MutableRefObject<AillyStoreDispatch>) {
 
   const reducers = {
     updateState(state: AillyPageState): AillyPageState {
-      return {
+      const newState = {
         story,
         instruction,
         selections: [...selections.map((opts) => [...opts])],
         storyItem,
         response: { ...response },
       };
+      return newState;
```

Comment:
This was just me debugging.

Reply:
This whole thing was me on a plane and in Vegas /shrug